community/kernel/src/test/java/org/neo4j/kernel/impl/core/TestLoopRelationships.java

```diff
@@ -71,11 +71,11 @@ public void canDeleteNodeAfterDeletingItsRelationshipsIfThoseRelationshipsInclud
         txCreateRel( node );
         txCreateLoop( node );
 
-        node.delete();
         for ( Relationship rel : node.getRelationships() )
         {
             rel.delete();
         }
+        node.delete();
```

Comment:
Sure: it was calling methods on a deleted node which should have thrown exceptions, but didn't. I would say the previous behaviour was buggy.